internal/profile/profile_json.go

```diff
@@ -0,0 +1,55 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package profile
+
+import (
+	"encoding/json"
+	"os/user"
+	"path/filepath"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/version"
+)
+
+// Metadata stores the data associated with a given profile
+type Metadata struct {
+	Name        string
+	DateCreated time.Time `json:"date_created"`
+	User        string
+	Version     string `json:"elastic_package_version"`
```

Comment:
The `elastic_package_version ` field is presented in the CLI table as `VERSION`. Do you think we can make it consistent here (just `version`)?